optimum/neuron/utils/misc.py

```diff
@@ -532,7 +540,7 @@ def download_checkpoints_in_cache(
 
 
```

Comment:
@requires_torch_neuronx

Reply:
For `replace_weights`? actually it shall work for inf1 as well.

Reply:
Alright!